tallycash/extension
# Rename Addresses

This PR adds the ability for users to rename their addresses!

https://user-images.githubusercontent.com/94649004/164518501-3dcbbded-3d12-418d-bbd7-56eb7e6284a8.mov

Some notes:
- Transactions in the Activity List are not retroactively updated - this will require going back through every transaction associated with a given address and re-annotating it.
- Once names are set they cannot be removed - only changed.  I didn't see much point of letting users go back to their arbitrarily assigned tally wallet names.


### To Test
- Change an addresses name as per the above video.
- Confirm that the name changes propagate throughout the application (with the above caveat in mind).
